src/components/AddBook.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable */
+import { useDispatch } from 'react-redux';
+import { uuid } from 'uuidv4';
+import { addBook } from '../redux/books/books';
+import React, { useState } from 'react';
+
+const AddBook = (props) => {
+  const [data_Title, set_Title] = useState('');
+  const [data_Author, set_Author] = useState('');
+
+  const dispatch = useDispatch();
+  const handleSubmit = e => {
+    e.preventDefault();
+    const newBook = {id: uuid(), title: data_Title, author: data_Author};
+    set_Title('');
+    set_Author('');
+    dispatch(addBook(newBook))
+  }
+  return (
+    <div className='formContainer'>
+      <form className='form' onSubmit={handleSubmit}>
+        <label>Add a new book</label>
+        <input className='input' onChange={(e) => set_Title(e.target.value)} value={data_Title} type='text' placeholder='Title'/>
+        <input className='input' onChange={(e) => set_Author(e.target.value)} value={data_Author} type='text' placeholder='Author'/>
+        <button type='submit' className='submitBtn'>Submit</button>
+      </form>
```

Comment:
- [ ] Great work on imlementing the requirements for this project, but as at now the user is unable to add a new book.
![image](https://user-images.githubusercontent.com/81932478/202042347-dc31e0f2-01ea-4de4-b51c-0bcc8669c294.png)
- [ ] The submit button does not work, kindly fix it

Reply:
Thank you! Submit button now working properly